.gitattributes

```diff
@@ -1,2 +1,3 @@
 *.csv.gz filter=lfs diff=lfs merge=lfs -text
 *.sqlite filter=lfs diff=lfs merge=lfs -text
+*.npy filter=lfs diff=lfs merge=lfs -text
```

Comment:
also, let's keep this repo nimble and avoid adding `.sqlite` and `.csv.gz` files (for now)

Reply:
@gwaybio 

These file types are already added to wave 1. I can make sure to git ignore any .sqlite files and .csv.gz files for wave 2.

Do you want me to go back and remove these files from wave 1?